Sources/Convenience/Optional.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Optional.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 11/05/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+
+@available(OSX 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+public extension Optional {
+    /// A method form of `Optional.Publisher.init`, bringing `Optional` on the same footing with `Result.publisher`.
```

Comment:
I'd use the same description as the original initialiser, I don't think docs should necessarily expose implementation details but mainly "what does this do"
```suggestion
    /// A publisher to emit the optional value of a successful result, or fail with an error.
```